lib/camt_parser/xml.rb

```diff
@@ -9,7 +9,7 @@ def self.parse(doc)
       case namespaces
       when 'urn:iso:std:iso:20022:tech:xsd:camt.052.001.02'
         return CamtParser::Format052::Base.new(doc.xpath('Document'))
-      when 'urn:iso:std:iso:20022:tech:xsd:camt.053.001.02'
+      when /urn:iso:std:iso:20022:tech:xsd:camt\.053\.001\.0(?:2|4)/
```

Comment:
Wondering if just adding both cases here wouldn't actually make it more readable.
It's only string comparison with a limited set after all.